app/views/shared/_landing_page_filter.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="organisation-container">
+  <ul class="govuk-list" data-cy="organisation-filter-list">
+    <% @organisations.each do |organisation| %>
+      <div class="govuk-grid-column-one-third filter-card">
+        <li class="organisation-item">
+          <a class="govuk-link <%=organisation.name%>-filter-link" href="<%= data_services_path(filters: organisation.id) %>" data-cy="<%= organisation.name%>-filter-link">
+            <% if organisation.name == 'Department for Work and Pensions' %>
+              <%= render partial: 'shared/dwp_logo', locals: { organisation: organisation }, class: "govuk-image", alt: "Department for Work and Pensions" %>
+            <% elsif File.exist?("app/assets/images/#{organisation.name.gsub(' ', '_')}.png") %>
```

Comment:
Name your files accordingly, these `gsub`s are easy to avoid

Reply:
What happens if file doesn't exist and it's not DWP? Is an `else` enough there?

Reply:
With the file naming, are you suggesting to rename them as camelcase like 'FoodStandardsAgency.png?' I think we would need some kind of name conversion logic anyway as we are comparing the organisation name in the database to the image file name; the db name would have spaces in between the words but you shouldn't have spaces in the image file name.

Reply:
(Even if the logic belongs in a presenter/helper)